docs/docs-content/clusters/palette-virtual-clusters/configure-oidc-virtual-cluster.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_label: "Configure OIDC for a Virtual Cluster"
+title: "Configure OIDC for a Virtual Cluster"
+description: "Learn how to configure OIDC for Palette Virtual Clusters."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["clusters", "cluster groups", "virtual clusters"]
+---
+
+
+Before deploying a virtual cluster, you must set up OpenID Connect (OIDC) Identity Provider (IDP) for the cluster group the virtual cluster belongs to. As virtual clusters are deployed to that cluster group, OIDC information in the cluster group configuration will be inserted into the virtual cluster configuration. 
+
+
+## Prerequisites
+
+- A healthy host cluster that you will use to create a cluster group.
+- A cluster group to which you can add virtual clusters.
+- An issuer URL and client ID that you obtain from your identity provider.
+- Privileges sufficient to edit cluster group settings and a viewer role to access the generated kubeconfig. If you are deploying virtual clusters, you will need admin privileges.
```

Comment:
Let's specify the Palette RBAC permission.  For the Kubeconfig, let's use `cluster.delete` permission as the minimum requirement but send them to this [page](https://docs.spectrocloud.com/clusters/cluster-management/kubeconfig/#kubeconfig-files) for more context.

```
clusterGroup.update
```

Reply:
applied